packages/core/lib/console.js

```diff
@@ -118,7 +122,6 @@ class Console extends EventEmitter {
     });
 
     this.resetContractsInConsoleContext(abstractions);
```

Comment:
Would you consider renaming this function to `addContractReferencesToReplContext`?

Reply:
I think this is a reset and not an addition. It happens at the end of every command, and if the contracts have changed they get reset to the new ones.  Happy to discuss further but this was the original name if this function in the replManager and I think it is still apt.